### ISSUE TITLE
Issues using DeepExplainer with Inception v3

### ISSUE THREAD

**Issue by gokul-uf** [shap/shap#800]
Hi,
I am trying to use the `DeepExplainer` class with a pretrained inception v3 architecture from [pytorch hub](https://pytorch.org/hub/pytorch_vision_inception_v3/). 
I get a lot of the following warnings
```
Warning: unrecognized nn.Module: InceptionE
Warning: unrecognized nn.Module: InceptionE
Warning: unrecognized nn.Module: InceptionE
Warning: unrecognized nn.Module: InceptionE
Warning: unrecognized nn.Module: InceptionE
Warning: unrecognized nn.Module: InceptionE
Warning: unrecognized nn.Module: InceptionD
Warning: unrecognized nn.Module: InceptionD
Warning: unrecognized nn.Module: InceptionD
Warning: unrecognized nn.Module: InceptionC
Warning: unrecognized nn.Module: InceptionC
Warning: unrecognized nn.Module: InceptionC
Warning: unrecognized nn.Module: InceptionC
Warning: unrecognized nn.Module: InceptionC
Warning: unrecognized nn.Module: InceptionC
Warning: unrecognized nn.Module: InceptionC
Warning: unrecognized nn.Module: InceptionC
Warning: unrecognized nn.Module: InceptionC
Warning: unrecognized nn.Module: InceptionC
Warning: unrecognized nn.Module: InceptionC
Warning: unrecognized nn.Module: InceptionC
Warning: unrecognized nn.Module: InceptionB
Warning: unrecognized nn.Module: InceptionB
Warning: unrecognized nn.Module: InceptionB
Warning: unrecognized nn.Module: InceptionA
Warning: unrecognized nn.Module: InceptionA
Warning: unrecognized nn.Module: InceptionA
Warning: unrecognized nn.Module: InceptionA
Warning: unrecognized nn.Module: InceptionA
Warning: unrecognized nn.Module: InceptionA
Warning: unrecognized nn.Module: InceptionA
Warning: unrecognized nn.Module: InceptionA
Warning: unrecognized nn.Module: InceptionA
Warning: unrecognized nn.Module: BasicConv2d
Warning: unrecognized nn.Module: BasicConv2d
Warning: unrecognized nn.Module: BasicConv2d
Warning: unrecognized nn.Module: BasicConv2d
Warning: unrecognized nn.Module: BasicConv2d
Warning: unrecognized nn.Module: BasicConv2d
Warning: unrecognized nn.Module: BasicConv2d
Warning: unrecognized nn.Module: BasicConv2d
Warning: unrecognized nn.Module: BasicConv2d
Warning: unrecognized nn.Module: BasicConv2d
Warning: unrecognized nn.Module: BasicConv2d
Warning: unrecognized nn.Module: BasicConv2d
Warning: unrecognized nn.Module: BasicConv2d
Warning: unrecognized nn.Module: BasicConv2d
Warning: unrecognized nn.Module: BasicConv2d
```

I checked out other issues on the repo talking about similar issues but I am not sure how to fix my issue. Any help would be appreciated!

**Comment by gabrieltseng** [comment]
Hi! These warnings are just to let you know that the DeepExplainer doesn't recognize that module (i.e. that it wasn't explicitly told [here](https://github.com/slundberg/shap/blob/master/shap/explainers/deep/deep_pytorch.py#L361) how to handle the module).

When this is the case, the DeepExplainer doesn't change the gradient it receives.

All the modules which throw up a warning all consist of the [BasicCov2d](https://github.com/pytorch/vision/blob/master/torchvision/models/inception.py#L344) block, which itself consists of a Conv2d and a Batchnorm layer. Those two modules *are* handled by the DeepExplainer.

Which is all to say that the DeepExplainer should work fine for the Inception model, in spite of the warnings.
